### ISSUE TITLE
Manual Pvc Error 'Cannot set property ExecStopPost, or unknown property'

### ISSUE THREAD

**Issue by Neii** [yandex-cloud/k8s-csi-s3#56]
When following this tutorial to operate [https://github.com/yandex-cloud/k8s-csi-s3#static-provisioning](url) `--no-systemd ` options not working 

got this error
`E0531 05:37:28.119030 1 utils.go:101] GRPC error: Error starting systemd unit geesefs-manualbucket_2fpath.service on host: Cannot set property ExecStopPost, or unknown property.`

**Comment by vskubriev** [comment]
Sure, i faced same issue.

When utilizing static pv (without storage class). Or with `storageClassName: ""` i.e.: `# Empty storage class disables dynamic provisioning`

There is no way (more likely) to setup static persistent volumes with option `--no-systemd`

**Comment by vskubriev** [comment]
Here is a link to the code
https://github.com/yandex-cloud/k8s-csi-s3/blob/519c4f0bd7f661bd09cc2f4b20b460c2eb26cd77/pkg/mounter/geesefs.go#L105

As I understand there is no way to disable systemd in our case (

**Comment by Neii** [comment]
Yeah I guess it might be a compatibility issue, try this release version `v0.35.4` . In my testing,  mount static PV success,  i don't add `--no-systemd` options. The disadvantage of the old version is that it may not be so elegant when ‘cleanup’. 
See here:
https://github.com/yandex-cloud/k8s-csi-s3/commit/96818e563aac7779e8799fe220ace2108cc659de

**Comment by Neii** [comment]
Or this way?
https://github.com/yandex-cloud/k8s-csi-s3/pull/34/files

**Comment by vskubriev** [comment]
> 

Thank you very much for help. I tested version `v0.35.4` and found that it works out of the box with systemd like a charm.

So I think this issues should not be closed as issue describe by me above in the ticket with systemd exists.

**Comment by vskubriev** [comment]
> Or this way? https://github.com/yandex-cloud/k8s-csi-s3/pull/34/files

Amazing flow. It seemed to me that this would not work. But I did not look in the code whether this parameter is processed or not. That's why I didn't try this trick. But I will keep it in mind. Thanks a lot again.

**Comment by vitalif** [comment]
Hi, maybe this is related to your system version?
https://www.freedesktop.org/wiki/Software/systemd/dbus/ shows that the dbus interface does have ExecStopPost
